delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -0,0 +1,465 @@
+                        'type': 'EquipmentAlarm',
+                        'sequence_number': alert_map.get('event-code'),
+                        'occur_time': occur_time,
+                        'description': alert_name,
+                        'resource_type': resource_type,
+                        'location': location
+                    }
+                    alert_list.append(alert_model)
+            return self.repeated_err(alert_list)
+        except Exception as e:
+            err_msg = "Failed to get storage error: %s" % (six.text_type(e))
+            LOG.error(err_msg)
+            raise e
+
+    def repeated_err(self, detail_info):
+        exist_questions = set()
+        detail_arr = []
+        for item in detail_info:
+            question = item['resource_type']
+            if question not in exist_questions:
+                exist_questions.add(question)
+                detail_arr.append(item)
+        return detail_arr
+
+    @staticmethod
+    def parse_alert(alert):
+        try:
+            alert_model = dict()
+            alert_id = ''
+            description = ''
+            severity = SSHHandler.TRAP_SEVERITY_MAP.get('8')
+            seq_num = ''
+            event_type = ''
+            for alert_key, alert_value in alert.items():
+                if SSHHandler.OID_ERR_ID in alert_key:
+                    alert_id = str(alert_value)
+                elif SSHHandler.OID_EVENT_TYPE in alert_key:
+                    event_type = alert_value
+                elif SSHHandler.OID_EVENT_DESC in alert_key:
+                    description = alert_value
+                elif SSHHandler.OID_SEVERITY in alert_key:
+                    severity = SSHHandler.TRAP_SEVERITY_MAP\
+                        .get(alert.get(SSHHandler.OID_SEVERITY),
+                             constants.Severity.INFORMATIONA)
+                elif SSHHandler.OID_EVENT_ID in alert_key:
+                    seq_num = alert_value
+            if len(description) > 0:
+                desc_arr = description.split(",")
+                if desc_arr:
+                    alert_id = SSHHandler.handle_split(desc_arr[0], ":", 1)
+                    if len(desc_arr) > 2:
+                        description = desc_arr[1]
+            alert_model['alert_id'] = str(alert_id)
+            alert_model['alert_name'] = event_type
+            alert_model['severity'] = severity
+            alert_model['category'] = constants.Category.FAULT
+            alert_model['type'] = constants.EventType.EQUIPMENT_ALARM
+            alert_model['sequence_number'] = seq_num
+            now = time.time()
+            alert_model['occur_time'] = int(round(now * SSHHandler.
+                                            SECONDS_TO_MS))
+            alert_model['description'] = description
+            alert_model['resource_type'] = ''
+            alert_model['location'] = description
+            print(alert_model)
```

Comment:
`print` syntax should be deleted in product environment.

Reply:
Deleted print